### DISCUSSION TITLE
Sort inside a list

### DISCUSSION THREAD

**Discussion by qqpann** [community/community#8908] [Other Feature Feedback, Questions, & Ideas]
The stared repositories are sorted by the added date.
However repositories in a list is not sorted. 
Options to sort it by date or name would be great.

**Comment by Kristine1975** [comment]
How about a list has the same filter options as stars already have: 
<img width="915" alt="image" src="https://user-images.githubusercontent.com/11298402/147480245-e1a95543-50e9-4b32-bf95-20da23b95444.png">

**Reply by bsshenrique** [reply]
He said _sort **inside** a list_

I agree.
An option when saving/editing lists with something like:
Order this list by: "stars count | date added | name"
Will be great.

**Reply by Kristine1975** [reply]
That's what I meant: sorting and filtering the repositories *inside a list*. While right now you can filter/sort all of your starred repositories, you can't do so inside a list. It would be useful for example to be able to sort them by "last activity", or filter them by programming language.
